sdk/test-utils/recorder/tsconfig.json

```diff
@@ -1,9 +1,15 @@
 {
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "declarationDir": "./types",
-    "outDir": "./dist-esm",
-    "lib": ["dom"]
+    "lib": ["dom"],
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "rootDir": "."
   },
-  "include": ["./src/**/*.ts", "./test/**/*.ts"]
+  "include": [
+    "./src/**/*.ts",
+    "./test/**/*.ts",
+    "src/utils/encoding-browser.mts",
```

Comment:
Why are these in the includes?  Shouldn't we just have the following?

```
  "include": [
    "./src/**/*.ts",
    "./src/**/*.mts",
    "./src/**/*.cts",
    "./test/**/*.ts",
    "./samples-dev/**/*.ts"
  ]
```

Reply:
updated